header.html

```diff
@@ -145,19 +148,20 @@ <h2>About Myself</h2>
                     <li class="p hr">Skills <img src="img/Union-right.svg" alt="counter" class="counter"></li>
                 </ul>
                 </div>
-            </section>
-            <!-- Contact form -->
-            <section class="form-group">
-                <img src="img/contact-form-shapes-mobile.svg" alt="Background shapes design" class="form-bg-img">
+        </section>
+        <!-- Contact form -->
+        <section class="form-group">
+            <div class="form-child">
```

Comment:
Pleadse check the padding of `"form-child" <div>` and try to make it closer to the Figma design.